RELEASES.md

```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+## 0.1.0
+
+### Changed
+
+- renamed crate from `stretch2` to `sprawl`
+
+### Removed
+
+- removed Javascript / Kotlin / Swift bindings
+  - the maintainer team lacks expertise to keep these working
+  - more serious refactors are planned, and this will be challenging to keep working through that process
+  - if you are interested in helping us maintain bindings to other languages, get in touch!
```

Comment:
We could link the GitHub Discussions here (assuming the repo will be moved to DioxusLabs/sprawl):
```suggestion
  - if you are interested in helping us maintain bindings to other languages, [get in touch](https://github.com/DioxusLabs/sprawl/discussions)!
```